contracts/TokenSale.sol

```diff
@@ -106,9 +105,13 @@ contract TokenSale is FinalizableCrowdsale, Pausable {
         crowdsaleCap = _crowdsaleCap.mul(10 ** 18);
 
         if (isMinting) {
+            require(tokenOwnerAfterSale != address(0), "TokenOwnerAftersale cannot be empty when minting tokens!");
             require(ERC20Plus(tokenOnSale).paused(), "Company token must be paused upon initialization!");
+        } else {
+            require(tokenOwnerAfterSale == address(0), "TokenOwnerAftersale cannot be empty when minting tokens!");
```

Comment:
Same error message as in the first if block message. It should read that address must be zero